js/src/3rdparty/etherscan/util/address.js

```diff
@@ -0,0 +1 @@
+../../../abi/util/address.js
```

Comment:
Ok, so why is the symlink not `../../../api/util/something` ?

In all honestly, the address stuff does not actually belong in the ABI, it just "landed" there.